mastodon.go

```diff
@@ -76,6 +77,7 @@ func (c *Client) doAPI(ctx context.Context, method string, uri string, params in
 		}
 	}
 	req = req.WithContext(ctx)
+	req.Header.Set("Accept-Encoding", "gzip")
```

Comment:
gzip,deflate

Reply:
Deflate is handled by other compression function, and also ress reliable than gzip.

Servers not supporting gzip is less likely to support deflate as well